### ISSUE TITLE
Look into mods to help structure progression

### ISSUE THREAD

**Issue by UndeadZeratul** [Nincraft/NincraftElectricBoogaloo2RevengeoftheFallen#271]
Something to help structure and guide the player through the pack, show them what they have access to, and where to go.  Since we edit so much of the pack's progression, it'd be nice to be able to see at a glance what I can do and where I can go from here.

Possible options include:
HQM: http://www.curse.com/mc-mods/minecraft/hardcore-questing-mode
Progression: http://www.curse.com/mc-mods/minecraft/229495-progression
Eureka: http://www.curse.com/mc-mods/minecraft/222995-eureka

**Comment by Nincodedo** [comment]
I want to avoid gating anything with these mods. The only structure we should add is a guide. Basically here are the materials you have, this is what you can build now, and if you got a few of these other materials you don't have, here's what you could make. By the way, here's how you go about getting those other materials.

I think HQM could greatly help with this, however I would not want to give out any rewards. HQM for us would just be a guide book, not a quest book, so I'm not sure this is 100% the best option.

**Comment by UndeadZeratul** [comment]
I completely agree, HQM would only really be added as a progression guide, rather than the "standard" that other packs such as Agrarian Skies or Regrowth have been using as.

That isn't to say quests wouldn't have a reason to be completed, it's just that we shouldn't hand out free resources for gathering X amount of the same resource, and adding random loot bags either breaks progression if RNG plays in your favor, or is unrewarding because the loot drops from them are boring and unnecessary.  There either needs to be a reason to use the quests, or make it perfectly clear they're not meant to be quests, but rather a visual guide for how to progress, and ease players into what they can do at any point in the pack, given their current situation.

**Comment by UndeadZeratul** [comment]
Updated OP to reflect this, but Enchiridion has a feature wherein we can write up huge library pages with various information, including things like how to survive the first night, where to get certain ores, how to progress further into the pack, etc.
